packages/Switches/src/index.tsx

```diff
@@ -0,0 +1,136 @@
+import withUtils from "@blaze-react/utils";
+import React, { FunctionComponent, useState } from "react";
+
+interface IOptions {
+  checked: boolean;
+  value: string;
+  disabled: boolean;
+  required: boolean;
+  label: string;
+  id?: string;
+}
+interface ISwitchesProps {
+  labelPosition?: string;
+  options: IOptions[] | IOptions;
```

Comment:
If you have a default prop then options should be optional or instead delete the default props